adagrams/game.py

```diff
@@ -1,11 +1,93 @@
+import random 
+
+available_letters = {
+    'A': [9, 1],
+    'B': [2,3], 
+    'C': [2, 3], 
+    'D': [4, 2], 
+    'E': [12, 1], 
+    'F': [2, 4], 
+    'G': [3, 2], 
+    'H': [2, 4], 
+    'I': [9, 1], 
+    'J': [1, 8], 
+    'K': [1, 5], 
+    'L': [4, 1], 
+    'M': [2, 3], 
+    'N': [6, 1], 
+    'O': [8, 1], 
+    'P': [2,3], 
+    'Q': [1,10], 
+    'R': [6,1], 
+    'S': [4,1], 
+    'T': [6,1], 
+    'U': [4,1], 
+    'V': [2,4], 
+    'W': [2,4], 
+    'X': [1,8], 
+    'Y': [2,4], 
+    'Z': [1,10]
+}
 def draw_letters():
-    pass
+    """Draws a hand of 10 letters"""
+    
+    letter_hand = []
+
+    while len(letter_hand) < 10:
+        picked_letter = random.choice(list(available_letters.keys()))
```

Comment:
Calling list() on a dictionary takes all the keys and puts them in a list so we get a list of 26 letters here.

In order to select a random letter from a pool that reflects what you have declared in `available_letters`, you could create a list pool_of_letters and use the keys and values from `available_letters` to populate a list with 9 As, 2 Bs, 2 C,s etc.

A suggestion could be:

```python
pool_of_letters = []
for letter, number in available_letters.items():
        pool_of_letters += letter * number

# You could then do user `random.choice(pool_of_letters)` 
```